webpack/components/QuestionForm.js

```diff
@@ -303,6 +317,11 @@ class QuestionForm extends Component{
     });
   }
 
+  actionWord() {
```

Comment:
Waiting on which of the feedback from the usability stuff we are going to adopt into backlog items.